main/webapp/modules/core/scripts/dialogs/expression-preview-dialog.js

```diff
@@ -350,15 +350,30 @@ ExpressionPreviewDialog.Widget.prototype._renderStarredExpressions = function(da
         var o = Scripting.parse(entry.code);
         
         $('<a href="javascript:{}">'+$.i18n('core-dialogs/remove')+'</a>').appendTo(tr.insertCell(0)).click(function() {
-            Refine.postCSRF(
-                "command/core/toggle-starred-expression",
-                { expression: entry.code, returnList: true },
-                function(data) {
-                    self._renderStarredExpressions(data);
-                    self._renderExpressionHistoryTab();
-                },
-                "json"
-            );
+            var removeExpression = DialogSystem.createDialog();
+                removeExpression.width("250px");
+            var removeExpressionHead = $('<div></div>').addClass("dialog-header").text($.i18n('core-dialogs/remove') + " " + $.i18n('core-dialogs/expression') + "?")
```

Comment:
Thanks for your attempt to reuse existing translations! However, we cannot do something like `$.i18n('core-dialogs/remove') + " " + $.i18n('core-dialogs/expression') + "?"` because that assumes that word order and punctuation remain identical in all languages. For instance in German you want to have the noun before the verb in this context. Or in French you want to have a space before the `?` and also an article before "expression". So you should introduce a new bit of UI text in this context. You can add it to the English file only, and translators will take care of adding it to other languages.